lib/util.js

```diff
@@ -170,6 +170,31 @@ module.exports.clone = function(obj) {
   return JSON.parse(JSON.stringify(obj));
 };
 
+module.exports.makeFakeZssPlugin = function() {
+  const zssFakePluginDef = {
+    "identifier": "org.zowe.zss.swagger",
+    "apiVersion": "1.0.0",
+    "pluginVersion": "1.0.0",
+    "pluginType": "application",
+    "swagger":[
+      {
+        "name" : "fileapi",
+        "version": "1.0.0"
+      },
+      {
+        "name" : "security-mgmt-api",
+        "version": "1.0.0"
+      },
+      {
+        "name" : "server-plugins-api",
+        "version": "1.0.0"
+      }
```

Comment:
I think this is okay for now, but shouldn't this be not hard-coded? We have a directory location for App server Swagger
zlux\zlux-app-server\doc\swagger

So shouldn't this array be generated by ZSF in swagger-catalog.js ?
We read them in `getSwaggerDocs(plugin, productCode, nodeContext, hostsContext)` method as you saw, so why are we hardcoding here? It seems like util.js just needs some info from swagger-catalog.js

Reply:
![image](https://user-images.githubusercontent.com/4791635/115299221-83fd9880-a12c-11eb-8caf-599dd2a4f9d2.png)

in normal plugin - we have to give dataService a `name`, but i could not use dataService property in fake plugin, it breaks as we require fake node express app for that.

used madeup property swagger, so i am just assigning name equivalent of `data` in above image, or giving name of service not of file.

getSwaggerDocs => assume swagger file name of service is same as dataservice name.

Reply:
Reference code- 
We make document name based on service name -> serviceName.{yaml,json} etc
https://github.com/NakulManchanda/zlux-server-framework/blob/3097ccd3472aee8a42a523e15550d753ced614b1/lib/swagger-catalog.js#L147
So, filename/dataservice name is provided in pluginDefinition.json 
`getSwaggerDocs` load files which match the service name only - not all files, so we have to provide the name in our plugin config explicitly.